cs481-ekh/f21-welolo
# Task #23

Verification Steps:
1. Execute build.sh
2. Navigate to both the server and client folders in two separate terminals
3. Run npm start in both the server and client folders
4. Open your browser and navigate to localhost:3000/
5. Enter '+17025763855' into the recipient text box
6. Enter an arbitrary message into the search box
7. Press Send Message, and monitor the Networks tab in the developer console for a 'send_message' request to localhost:3000/send_message
8. Verify the response is "{"success":true}"
9. I will verify that I received your text message
